src/PipeLineTemplate/lib/notification.ts

```diff
@@ -0,0 +1,29 @@
+import * as cdk from "@aws-cdk/core";
+import * as chatbot from '@aws-cdk/aws-chatbot';
+import * as ssm from '@aws-cdk/aws-ssm';
+import { BuildOperationsDetails } from "./buildOperationsDetails";
+import { IPipeline } from "@aws-cdk/aws-codepipeline";
+
+export class Notification extends cdk.Construct {
+    constructor(scope: cdk.Construct, id: string, props: BuildOperationsDetails, pipeline: IPipeline) {
+        super(scope, id);
+        
+        let slackChannelName = 'sphinx-env-';
+        if(props.githubRepositoryBranch === 'master' || props.githubRepositoryBranch === 'bumastemra-uat'){
```

Comment:
Can we change the second part to `props.githubRepositoryBranch.startsWith('bumastemra')`

That will cover `bumastemra` (PROD) also.